### PR TITLE
Do not init new project when install command is executed

### PR DESCRIPTION
**Pull request by Fatme**

### CODE REVIEW

**Comment by dtopuzov** [comment]
:100:

**Comment by rosen-vladimirov** [comment]
https://github.com/NativeScript/nativescript-cli/blob/fatme/install/lib/commands/install.ts#L4 - this line should be removed, it is not used anymore

File: lib/commands/install.ts [hunk 1]
```diff
@@ -9,8 +9,10 @@ export class InstallCommand implements ICommand {
 	constructor(private $fs: IFileSystem,
 		private $errors: IErrors,
 		private $logger: ILogger,
```
Comment (rosen-vladimirov) [inline lib/commands/install.ts]: `fs`, `errors` and `logger` are not used anywhere

File: lib/commands/install.ts [hunk 2]
```diff
@@ -9,8 +9,10 @@ export class InstallCommand implements ICommand {
...
-		private $options: IOptions,
-		private $injector: IInjector,
+		private $platformsData: IPlatformsData,
+		private $platformService: IPlatformService,
+		private $projectData: IProjectData,
+		private $projectDataService: IProjectDataService,
 		private $staticConfig: IStaticConfig) { }
```
Comment (rosen-vladimirov) [inline lib/commands/install.ts]: staticConfig is not used anywhere

**Comment by rosen-vladimirov** [comment]
Maybe this is out of the scope for current PR, but I've tried using `*` for tns-android:
```JSON
{
        "nativescript": {
                "id": "org.nativescript.app1004",
                "tns-android": {
                        "version": "*"
                }
        },
        "dependencies": {
                "lodash": "^3.9.3"
        }
}
```
And the result is:
```
Copying template files...
Failed to retrieve data from npm. Please try again a little bit later.. Error: Error: ENOENT, stat 'C:\Users\vladimirov\AppData\Roaming\npm-cache\tns-android\*\package'
```
Same happened when I tried `^1.1.0`

**Comment by teobugslayer** [comment]
@rosen-vladimirov -definitely outside the scope of this PR. I think we should support such versions but we may wait until the need raises.

**Comment by Fatme** [comment]
I totally agree with @teobugslayer .

**Comment by rosen-vladimirov** [comment]
:+1:  after green build

**Pull request merged**